docs/man_pages/bodhi.rst

```diff
@@ -179,6 +179,11 @@ The ``updates`` command allows users to interact with bodhi updates.
 
         A path to a file containing all the update details.
 
+    ``--requirements <Taskatron tasks>``
```

Comment:
It's spelled Taskotron (though the [cool looking] logo font makes it a little ambiguous): https://taskotron.fedoraproject.org/